docs/pywbemclicommands.rst

```diff
@@ -166,18 +166,106 @@ Thus, for example an ``instance enumerate`` with and without the ``-o`` option:
 
     root/cimv2:CIM_Foo.InstanceID="CIM_Foo3"
 
+.. _`Specifying INSTANCENAME`:
+
+Specifying INSTANCENAME
```

Comment:
How about "Specifying INSTANCENAME command argument"

Reply:
DONE; Changed to "Specifying the INSTANCENAME command argument"